packages/ui-toolkit/src/components/atoms/Button/Button.tsx

```diff
@@ -121,7 +123,8 @@ class Button extends React.PureComponent<Props> {
     loadingText: 'Loading...',
     fontSize: '',
     textColor: '',
-    backgroundColor: ''
+    backgroundColor: '',
+    dataTestId: ''
```

Comment:
why are we initialising this with a default value when there is no need for that , why can't we just mark this as an optional prop and add a check on dataTestId instead of dataTestId.length